site/homepage/layouts/partials/head.html

```diff
@@ -60,25 +57,56 @@
   <meta name="msapplication-config" content="{{ "/favicon/browserconfig.xml" | relURL }}">
   <meta name="theme-color" content="#ffffff">
 
-  <!-- owl carousel css -->
-
+  <!-- owl carousel CSS -->
   <link href="{{ "css/owl.carousel.css" | relURL}}" rel="stylesheet">
   <link href="{{ "css/owl.theme.css" | relURL}}" rel="stylesheet">
 
-  <link rel="alternate" href="{{ .Site.BaseURL }}/index.xml" type="application/rss+xml" title="{{ .Site.Title }}">
+  <!-- RSS feed -->
+  <link rel="alternate" href="{{ "/index.xml" | absURL }}" type="application/rss+xml" title="{{ .Site.Title }}">
```

Comment:
This resolves to `https://www.eclipse.org/index.xml` but should be `https://www.eclipse.org/hono/index.xml`. I think that this should do the trick:
`<link rel="alternate" href="{{ .Site.BaseURL }}index.xml" type="application/rss+xml" title="{{ .Site.Title }}">`